frontend/src/leaderboard.js

```diff
@@ -0,0 +1,31 @@
+const ACCOUNTS_URL = "http://localhost:3000/accounts";
+document.addEventListener('DOMContentLoaded', () => {
+  main();
+});
+function main() {
+  fetchUsers();
+}
```

Comment:
is main needed here? I have seen this pattern quite a bit and if it's just calling main on DOM Content loaded, I feel like you might as well just call the functions in the above.

Reply:
changed, thanks.